apps/intake/tests/utils/UploadDocs.ts

```diff
@@ -45,6 +45,21 @@ export class UploadImage {
     return uploadedPhoto;
   }
 
+  async uploadFile(locator: Locator, file: Locator): Promise<{ uploadedFile: Locator; link: string | null }> {
+    const __filename = fileURLToPath(import.meta.url);
+    const __dirname = dirname(__filename);
+
+    const [fileChooser] = await Promise.all([this.page.waitForEvent('filechooser'), locator.click()]);
+
+    const filePath = path.join(this.getPathToProjectRoot(__dirname), `/images-for-tests/template.pdf`);
+    await fileChooser.setFiles(filePath);
+    await this.page.waitForTimeout(5000);
```

Comment:
Better not to use waitForTimeout. Preferable options:
- Use UI state for track loading (loaders, elements, e.t.c)
- Or await a network request

Reply:
It's also possible to use a check with a timeout: `await expect(uploadedFile).toBeVisible({timeout: 15_000});`
This doesn't require any additional UI state tracking and will check automatically during the timeout period.